src/zcl_abapgit_repo_content_list.clas.abap

```diff
@@ -197,8 +197,12 @@ CLASS ZCL_ABAPGIT_REPO_CONTENT_LIST IMPLEMENTATION.
 
   METHOD filter_changes.
 
-    DELETE ct_repo_items WHERE changes = 0.
+    FIELD-SYMBOLS: <ls_item> TYPE zif_abapgit_definitions=>ty_repo_item.
 
+    DELETE ct_repo_items WHERE changes = 0.
+    LOOP AT ct_repo_items ASSIGNING <fs_item>.
```

Comment:
I don't think this compiles? `<fs_item>`

Reply:
Fixed.